doc/sphinx/source/introduction.rst

```diff
@@ -144,10 +135,10 @@ back.
 Debug console
 ^^^^^^^^^^^^^
 
-In widelands debug builds you can open a debug console by pressing ``F6``. You
-can enter Lua commands here that act in the global environment: That is if you
-are in a scenario you can access the global variables and alter all Lua
-objects that are in the global scope:
+In widelands debug builds you can open a debug console by pressing 
+``SHIFT+CTRL+SPACE``. You can enter Lua commands here that act in the global
```

Comment:
```suggestion
``Ctrl+Shift+Space``. You can enter Lua commands here that act in the global
```

Reply:
Done, except that i did not change the order of keys. For me it's handy to have this ordering.

Same for below.

Reply:
It's inconsistent with the UI though, where the ordering of modifier keys is always the same:
![grafik](https://user-images.githubusercontent.com/48293446/113488014-adc38800-94bb-11eb-9b0a-38e67b1a7426.png)